### DISCUSSION TITLE
Bookmarks icons are not displayed on the side

### DISCUSSION THREAD

**Discussion by kosh2323** [alefragnani/vscode-bookmarks#533] [Q&A] [answered]
Hello, the plugin creates bookmarks and you can see it, but the icons of these bookmarks are missing in the main window, why?Win10x64 21H2 LTSC.
![1](https://user-images.githubusercontent.com/82117170/164465153-82506068-2383-4c7c-a3c8-1cf23f8cf4ee.jpg)

**Comment by alefragnani** [comment, marked as answer]
Hi @kosh2323,

Maybe you are facing this issue (https://github.com/alefragnani/vscode-bookmarks/issues/280#issuecomment-662406813 ) caused by a VS Code setting which effectively hides glyphs from the gutter.

Please, take a look at your settings.

Hope this helps

**Reply by kosh2323** [reply]
> Hi @kosh2323,
> 
> Maybe you are facing this issue ([#280 (comment)](https://github.com/alefragnani/vscode-bookmarks/issues/280#issuecomment-662406813) ) caused by a VS Code setting which effectively hides glyphs from the gutter.
> 
> Please, take a look at your settings.
> 
> Hope this helps

Yes, thank you very much I had it turned off as soon as I did the "editor.glyphMargin": true everything worked

**Comment by fade2gray** [comment]
Doesn't seem to be the case in the image above, otherwise the line numbering would be hard against the left edge of the editor.

**Comment by alefragnani** [comment]
If that's not the case, I wonder where/how icons would be displayed on such tiny space left by the line numbers in the gutter.

**Reply by fade2gray** [reply]
Yes, you're correct, I think I ought to get my eyes tested. 😎

**Comment by kosh2323** [comment]
Yes, thanks a lot to everyone who wanted to help, I had it turned off as soon as I did the "editor.glyphMargin": true everything worked.
